signer/xpi/jar.go

```diff
@@ -169,3 +170,29 @@ func isSignatureFile(name string) bool {
 	}
 	return false
 }
+
+// mustReadFileFromZIP reads a given filename out of a ZIP and returns it or dies
+func mustReadFileFromZIP(signedXPI []byte, filename string) (data []byte) {
+	zipReader := bytes.NewReader(signedXPI)
+	r, err := zip.NewReader(zipReader, int64(len(signedXPI)))
+	if err != nil {
+		log.Fatalf("Error reading ZIP %s", err)
+	}
+
+	for _, f := range r.File {
+		if f.Name == filename {
+			rc, err := f.Open()
+			defer rc.Close()
+			if err != nil {
+				log.Fatalf("Error opening file %s in ZIP %s", filename, err)
+			}
+			data, err = ioutil.ReadAll(rc)
+			if err != nil {
+				log.Fatalf("Error reading file %s in ZIP %s", filename, err)
+			}
+			return
+		}
+	}
+	log.Fatalf("failed to find %s in ZIP", filename)
```

Comment:
why die? any particular reason? if not, returning an error seems much friendly to me.

Reply:
This is a test helper function I pulled out for use verifying client responses and 
`must` in the function name usually means the function must succeed.

Reply:
https://youtu.be/F_XaIuw6K6Q?t=7s

Reply:
fixed